NEWS.txt

```diff
@@ -3,20 +3,21 @@ Release Notes
 =============
 
 
-Theano 0.10.0beta2 (25th of August, 2017)
-=========================================
+Theano 0.10.0beta2 (5th of September, 2017)
+===========================================
 
 This release contains new features, improvements and bug fixes to prepare the upcoming release candidate.
 
 We recommend that every developer updates to this version.
 
 Highlights:
  - Support NumPy ``1.13``
+ - Added conda recipe
  - Optional faster optimization step with new destroy handler
  - Updated documentation
  - Bug fixes, crash fixes and warning improvements
 
-A total of 15 people contributed to this release since 0.10.0beta1, see list below.
+A total of 16 people contributed to this release since 0.10.0beta1, see list below.
```

Comment:
I don't like having a list of people only for between beta1 and beta2. Can you just keep the full list since 0.9? Do you have a reason to have such a specific list here?

Reply:
No particular reason, I used to list contributors between only the two last released versions.

I will update the list with contributors since 0.9.